### DISCUSSION TITLE
Regarding Storing wallet info for future send/receive money to the wallet using this code

### DISCUSSION THREAD

**Discussion by varun2india89** [swaponline/MultiCurrencyWallet#4140] [Q&A]
Hello, 
Please help me if I am not wrong. 
I need to create a new Token in the ERC20 and then add it here as new wallet right? 
if so the the Wallet hash generated for new coin or btc or eth have to be stored in Database So that, i will use it as wallet hash for ever, if some one send money from blockchain.info to this wallet, will it be received  ? and how it works. 
I need some help in setting up a wallet website with BTC, ETH, USDT, own coin, LTC, So people can register/login and then create required wallet and send/receive money to it which are verified in the blockchain explorer. Can you confirm how to do this or anyone can help me to do it ?

**Comment by Seyedehfatemehmir** [comment]
Drop your wallet logins details let's help you get through it
